src/script.js

```diff
@@ -839,7 +839,7 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 const addFloatingComposeButton = () => {
-  if (!document.querySelector('.floating-compose')) {
+  if (!select.composeButton()) {
```

Comment:
This is not correct - the previous code checks for the non-existence of the floating compose button that we added. This change checks for the non-existence of the original GMail compose button. As a result, our compose button does not get added

Reply:
Have submitted PR to revert the code

Reply:
Apologies for this. Upon closer inspection I have just found out that the floating compose button disappearance is caused by the uBlock Origin extension. The code that I sent corrected this issue, but, as you experienced, fails to render the button if uBlock is not connected.

The new solution that I have found, and seems to work regardless of uBlock being installed or not is:

`if (!document.querySelector('.floating-compose') || !select.composeButton())`
[https://github.com/team-inbox/inbox-reborn/compare/master...phicau:master](url)

I have just pushed this to my repo and will comment on the issues section to inform the users who may be affected. Despite not having experienced any side effects from the new change, since the issue is produced by a third party extension, I am not sure whether it makes sense sending the pull request/merging or not.

Reply:
I use uBlock origin, and have never experienced the issue